src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -167,11 +160,16 @@ public function setupFieldAssociations($fields, $table, $fieldConditions, $strat
             ]);
         }
 
+        $conditions = ["$targetAlias.model" => $model];
+        if (!$this->_config['allowEmptyTranslations']) {
+            $conditions["$targetAlias.content !="] = '';
+        }
```

Comment:
Sorry it's confusingly necessary for the PR I put in for this. I needed the field to not be returned to work nicely when using `filterByCurrentLocale`. Without the bit operation this would happen:
1. Table translated, `onlyTranslated` false
2. `Table->find('translations', ['filterByCurrentLocale' => true]);`
3. This returns records with empty non-null `content`.
4. Those records can't come back since they're not fully translated BUT the behavior can't be set with `onlyTranslated` true, since I need that to be false when working on a backend of possibly untranslated records.